src/tests/register-test.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../server';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Registration block test', () => {
+  it('register a user ', (done) => {
+    chai.request(app)
+      .post('/signUp')
+      .field({
+        email: 'pextech3333@gmail.com',
+        password: 'pextech',
+      })
+      .end((err, res) => {
+        res.body.should.be.a('object');
+        res.body.should.have.status(201);
```

Comment:
You need to add more test cases so that your tests cover all grounds. What I mean here is that your tests are always looking the same with just `res.body` which you're checking if it's an object and then the `res.status`. You're doing this over and over and over again.